keras/integration_test/distributed_training_test.py

```diff
@@ -77,7 +77,17 @@ def dataset_fn(input_context):
 
         x = tf.keras.utils.experimental.DatasetCreator(dataset_fn)
 
-        model.fit(x, epochs=2, steps_per_epoch=10)
+        model.fit(
+            x,
+            epochs=2,
+            steps_per_epoch=10,
+            callbacks=[
+                tf.keras.callbacks.TensorBoard(
+                    update_freq=5,
+                    write_steps_per_second=True,
+                )
+            ],
+        )
```

Comment:
Let's also add code to verify that the summaries are indeed written every 5 batches - you can refer to [1] for an example of how to verify this. Thanks!

[1] https://github.com/keras-team/keras/blob/4a670a3a5c4632f8f3ec8805fa2f49ccb1c89b13/keras/utils/sidecar_evaluator_test.py#L88-L95

Reply:
Do you want the tests here or in `callbacks_test.py`? I meant for the test in `distributed_training_test.py` to be more of a crash detector, especially since `update_freq` currently isn't working for all strategies

Reply:
`callbacks_test.py` may be more appropriate but the key is we'll need to test it on all strategies (and make sure that the unsupported strategies raise an error).